### PR TITLE
[IMP] Point of Sale: updated doc about pos for restaurant

### PR DESCRIPTION
**Pull request by Lau-Odoo**

### CODE REVIEW

**Comment by AntoineVDV** [comment]
tag ?

File: point_of_sale/restaurant/restaurant.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
+==================
+Manage your tables
+==================
+
+Restaurants have very specific needs that shops don’t have. That’s why
+Odoo gives restaurant owners several unique features to help them manage
+their business in the best possible way.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Restaurants have specific needs that shops don’t have. That’s why
Odoo gives restaurant owners several unique features to help them manage
their business in the best possible way.
```

This kind of suggestion comes from me reading the book Style, and he says we should not modify absolute words, unique, final or complete with very, more, quite and so on (:

File: point_of_sale/restaurant/restaurant.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Add a floor
+===========
+
+Once your *Point of Sale* has been configured, select *Table
+Management* in :menuselection:`Point of Sale --> Configuration -->
+Point of Sale`. Then, click on *Floors*.
+
+.. image:: media/restaurant_03.png
+    :align: center
+    :alt: View of the table management feature. Way to manage and create floors for a pos
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]: Reduce size of image?!

File: point_of_sale/restaurant/restaurant.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+To activate the bar/restaurant features, go to :menuselection:`Point of
+Sale --> Configuration --> Point of Sale` and open your PoS. Now,
+select *Is a Bar/Restaurant*.
+
+.. image:: media/restaurant_01.png
+    :align: center
+    :alt: Restaurant set up. Enabling the is a bar/Restaurant feature on a PoS
+
+Then, new features appear with a fork and a knife on the right, meaning
+that the features are restaurant-specific.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
New features are shown with a fork and a knife next to it to indicate
that they are restaurant-specific.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+.. image:: media/restaurant_02.png
+    :align: center
+    :alt: Several restaurant features. Bill printing, bill splitting and tips
+
+Add a floor
+===========
+
+Once your *Point of Sale* has been configured, select *Table
+Management* in :menuselection:`Point of Sale --> Configuration -->
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Management* under :menuselection:`Point of Sale --> Configuration -->
```

File: point_of_sale/restaurant/restaurant.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+Point of Sale`. Then, click on *Floors*.
+
+.. image:: media/restaurant_03.png
+    :align: center
+    :alt: View of the table management feature. Way to manage and create floors for a pos
+
+There, click on create and name your floor and tables.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion

```

File: point_of_sale/restaurant/restaurant.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+.. image:: media/restaurant_02.png
+    :align: center
+    :alt: Several restaurant features. Bill printing, bill splitting and tips
+
+Add a floor
+===========
+
+Once your *Point of Sale* has been configured, select *Table
+Management* in :menuselection:`Point of Sale --> Configuration -->
+Point of Sale`. Then, click on *Floors*.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Point of Sale`. Then, click on *Floors* to create and name your floor and tables.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+    :alt: Backend view of a restaurant floor. Table name and number of sits for each table
+
+.. note::
+   Don’t forget to link your floor to your point of sale.
+
+Add tables
+==========
+
+To add tables, you can also open your PoS interface to see your
+floor(s).
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
To add tables, open your PoS interface to see your
floor(s).
```

File: point_of_sale/restaurant/restaurant.rst [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+.. image:: media/restaurant_05.png
+    :align: center
+    :alt: View of the floors menu to manage several floors at the same time
+
+Then, click on the pencil to enter into edit mode. It will allow you to
+create tables, move them, modify them, …
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Then, click on *Edit Mode** (pencil icon on the upper right corner) to be allowed to
create, move, modify tables, etc.
```

*right name of the menu here

File: point_of_sale/restaurant/restaurant.rst [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+.. image:: media/restaurant_06.png
+    :align: center
+    :alt: View of the floor management. Add tables, the number of sits, their name and their shape
+
+.. note::
+   To make your table easier to find, you can rename them, change their
+   shape, size or even color. Adding the number of people they hold is also
+   possible.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
   To make your tables easier to be found, you can rename them, change their
   shape, size and even color. It is also possible to add the maximum number of sits the table can have.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+Register your table(s) orders
+=============================
+
+To register an order, click on a table. By doing so, you will be brought
+to your interface.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
To register an order, click on the respective table. By doing so, you are taken
to your main interface.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+.. image:: media/restaurant_07.png
+    :align: center
+    :alt: View of the pos interface to register orders
+
+Transfer customer(s)
+====================
+
+If your customers want to move to another table after they already
+ordered, use the transfer button so the order will be moved to the new
+table too.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
If your customers want to move to another table after they already
ordered, use the transfer button. This way the order is also moved to the new
table.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+
+To do so, select the table your customer is currently on.
+
+.. image:: media/restaurant_08.png
+    :align: center
+    :alt: View of the restaurant tables, one having a pending order
+
+Now, click on the transfer button and select the table you transfer your
+customer to.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Now, click on the transfer button and select the table to which you are transferring your
customer.
```

File: point_of_sale/restaurant/restaurant.rst [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+          to another
+
+Register an additional order
+============================
+
+When registering an order, use the + button to simultaneously proceed to
+another one.
+
+Then, you can move between each of your orders and process the payment
+when needed.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
Then, you can shift between your orders and process the payment
when needed.
```

File: point_of_sale/restaurant/restaurant.rst (hunk 1 above)
Comment (Lau-Odoo) [reply point_of_sale/restaurant/restaurant.rst]: Never had the chance to read Style, so I didn't know ^^
Thanks!

**Comment by Lau-Odoo** [comment]
Thanks Keli!

File: point_of_sale/restaurant/restaurant.rst [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+Manage your tables
+==================
+
+Restaurants have specific needs that shops don’t have. That’s why
+Odoo gives restaurant owners several unique features to help them manage
+their business in the best possible way.
+
+Floor and table management, bill splitting, or even the possibility to
+print the orders in the kitchen, everything is there to help your
+business to shine and your employees to work efficiently.
```
Comment (kbr-odoo) [inline point_of_sale/restaurant/restaurant.rst]:
```suggestion
print orders from the kitchen, everything is there to help your
business shine and your employees to work efficiently.
```

maybe 'from' the kitchen sounds better, no? ^^

File: point_of_sale/restaurant.rst [hunk 15]
```diff
@@ -5,11 +5,8 @@ Restaurant Features
...
-   restaurant/table
+   restaurant/restaurant
    restaurant/split
    restaurant/bill_printing
    restaurant/kitchen_printing
-   restaurant/tips
-   restaurant/transfer
-   restaurant/multi_orders
\ No newline at end of file
+   restaurant/tips
```
Comment (AntoineVDV) [inline point_of_sale/restaurant.rst]: Redirect rules

**Comment by Lau-Odoo** [comment]
Waiting for your review @jubodoo

**Comment by Lau-Odoo** [comment]
Still waiting for your review @jubodoo

**Pull request merged**